src/workspacer/Keybinds/KeybindManager.cs

```diff
@@ -375,45 +375,54 @@ private string GetKeybindString(KeyModifiers mods, Keys keys)
         {
             var parts = new List<string>();
 
+            if (mods.HasFlag(KeyModifiers.Win))
+                parts.Add("Win");
             if (mods.HasFlag(KeyModifiers.LAlt))
-                parts.Add("alt");
+                parts.Add("Alt");
             if (mods.HasFlag(KeyModifiers.LShift))
-                parts.Add("shift");
+                parts.Add("Shift");
 
             if (keys == Keys.Oemcomma) { parts.Add(","); }
             else if (keys == Keys.OemPeriod) { parts.Add("."); }
             else if (keys == Keys.Oem2) { parts.Add("/"); }
-            else if (new Regex("d\\d").IsMatch(keys.ToString().ToLower()))
+            else if (new Regex("d\\d").IsMatch(keys.ToString().Capitalize()))
```

Comment:
Currently because of the additional "d" in the regex this will print numbered keybinds as "DN" e.g. "D1"
I would change this to:
```csharp
else if(new Regex("\\d").IsMatch(keys.ToString().Capitalize()))
```

By removing the additional "d" it just returns the number which makes it cleaner IMO